### PR TITLE
 Add release notes for TP1.3(v0.14.0, v0.15.0)

### PR DESCRIPTION
**Pull request by rupalibehera**
# Changes

Adding release notes for TP1.3 which are pulled from upstream release v0.14.0 and v0.15.0
# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```
release-note
```

### CODE REVIEW

**Comment by rupalibehera** [comment]
cc: @savitaashture

**Review by vdemeester — APPROVED** [review]

File: openshift/release/release-notes.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+## Deprecation :broom:
+* remove --check shorthand and add global flags to version cmd https://github.com/tektoncd/cli/pull/1269
+
+# Tekton Cli v0.14.0
+
+## Features :sparkles:
+* Update OSX instruction to install from brew :beers: https://github.com/tektoncd/cli/pull/1196
+* Add EventListener Logs Command https://github.com/tektoncd/cli/pull/1192
+* Bump Knative/K8s dependencies https://github.com/tektoncd/cli/pull/1198
```
Comment (savitaashture) [inline openshift/release/release-notes.md:23]: i think we can remove `Bump Knative/K8s dependencies https://github.com/tektoncd/cli/pull/1198`

File: openshift/release/release-notes.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+* Refactor PipelineRun and TaskRun Cancel Error Based on Condition Status https://github.com/tektoncd/cli/pull/1189
```
Comment (savitaashture) [inline openshift/release/release-notes.md:24]: this also

File: openshift/release/release-notes.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+* Integrate Hub commands into tkn https://github.com/tektoncd/cli/pull/1203
+* Change --nocolour to --no-color https://github.com/tektoncd/cli/pull/1224
+* Updates Hub Dependency https://github.com/tektoncd/cli/pull/1221
```
Comment (savitaashture) [inline openshift/release/release-notes.md:27]: this one `Updates Hub Dependency`

File: openshift/release/release-notes.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+* Deprecate -c Shorthand for tkn version --check https://github.com/tektoncd/cli/pull/1232
+* Bump pipeline and triggers deps https://github.com/tektoncd/cli/pull/1225
```
Comment (savitaashture) [inline openshift/release/release-notes.md:29]: this also `Bump pipeline and triggers deps`

File: openshift/release/release-notes.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+* Add --all-namespaces flag to tkn triggerbinding/eventlistener/TriggerTemplate/condition list command
+  https://github.com/tektoncd/cli/pull/1234, https://github.com/tektoncd/cli/pull/1236, https://github.com/tektoncd/cli/pull/1235
+  and https://github.com/tektoncd/cli/pull/1241
+* Add --no-headers flag to tkn clustertask/clustertriggerbinding list command https://github.com/tektoncd/cli/pull/1244
+  and https://github.com/tektoncd/cli/pull/1243
+* Made sure to target LTS for ubuntus https://github.com/tektoncd/cli/pull/1238
+* Bump pipleine dependency to v0.18.0 https://github.com/tektoncd/cli/pull/1251
```
Comment (savitaashture) [inline openshift/release/release-notes.md:36]: this as well

File: openshift/release/release-notes.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+## Fixes :bug:
+* Common way of Referring to Tekton Resources in User Facing Messages: EventListener/TriggerBinding/ClusterTriggerBinding
+  /Condition/TriggerTemplate https://github.com/tektoncd/cli/pull/1208, https://github.com/tektoncd/cli/pull/1211,
+  https://github.com/tektoncd/cli/pull/1210, https://github.com/tektoncd/cli/pull/1209, https://github.com/tektoncd/cli/pull/1213
+* Avoid Use of Previous TaskRunSpecStatus and PipelineRunSpecStatus with Start Commands https://github.com/tektoncd/cli/pull/1201
+* tkn pr describe failing in pr with conditions https://github.com/tektoncd/cli/pull/1207
+* Fix Alignment of Usage https://github.com/tektoncd/cli/pull/1217
```
Comment (savitaashture) [inline openshift/release/release-notes.md:44]: this one `Fix Alignment of Usage`

File: openshift/release/release-notes.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+* Do Not Delete ClusterTask TaskRuns with --task Flag https://github.com/tektoncd/cli/pull/1227
+* Fix APIversion not shown on describe commands https://github.com/tektoncd/cli/pull/1237
+* fix function name typo https://github.com/tektoncd/cli/pull/1245
```
Comment (savitaashture) [inline openshift/release/release-notes.md:47]: this as well `fix function name typo`

**Review by savitaashture — COMMENTED** [review]
I assume below changes are not user facing we can remove

/cc @piyush-garg @chmouel @pradeepitm12

**Review by pradeepitm12 — COMMENTED** [review]
IMHO we can remove the prs as suggested by Savita, else is all good.

**Comment by rupalibehera** [comment]
> I assume below changes are not user facing we can remove
> 
> /cc @piyush-garg @chmouel @pradeepitm12

@savitaashture fixed your reviews I guess we are good to go now

**Review by vdemeester — APPROVED** [review]

**Pull request merged**